JS-test/main.js

```diff
@@ -0,0 +1,34 @@
+//Write a function that logs to the console numbers 1 to 100.
+function counting() {
+  for (let i = 1; i < 101; i++) {
```

Comment:
```suggestion
  for (let i = 1; i <= 100; i++) {
```

It's a small detail, but you can help readers out by using the `<=` operator. "loop over the numbers between 1 and 100" is easier than "loop over the numbers between 1 and less than 101".